src/openrct2/actions/LandSetHeightAction.cpp

```diff
@@ -360,7 +360,7 @@ money32 LandSetHeightAction::GetSurfaceHeightChangeCost(SurfaceElement* surfaceE
     {
         int32_t cornerHeight = tile_element_get_corner_height(surfaceElement, i);
         cornerHeight -= map_get_corner_height(_height, _style & TILE_ELEMENT_SURFACE_SLOPE_MASK, i);
-        cost += MONEY(abs(cornerHeight) * 5 / 2, 0);
+        cost += ToGBP(static_cast<long double>(abs(cornerHeight) * 5 / 2));
```

Comment:
the 10 here really represents the cost of modifying the height by a unit. I think `2.5_GBP * abs(cornerHeight)` would be just as good and much more descriptive.

Reply:
Nice suggestion!